node/mo/pages/api/links/[id].ts

```diff
@@ -10,7 +10,8 @@ const handler = async (req: NextApiRequest, res: NextApiResponse) => {
     const catcher = (error: Error) => res.status(400).json({ error })
 
     // GRAB ID FROM req.query (where next stores params)
-    const id: string = req.query.id as string
+    const id: string = decodeURIComponent(req.query.id as string)
```

Comment:
Have to encode URIs and decode them - so that we can add regexs and have valid urls that the router can handle.